composer.json

```diff
@@ -47,7 +47,7 @@
     "doctrine/doctrine-bundle": "^1.0",
     "monolog/monolog": "~1.8",
     "symfony/proxy-manager-bridge": "~2.4",
-    "phpunit/phpunit": "^4.8",
+    "phpunit/phpunit": "^5.7",
```

Comment:
is there a specific reason for requiring 5.7 and not 5.0?

Reply:
Nope, you're right